pkg/release/config.go

```diff
@@ -58,6 +58,8 @@ type config struct {
 	Wait                     bool `yaml:"wait,omitempty" json:"wait,omitempty" jsonschema:"description=Whether to wait for all resource to become ready,default=false"`
 	WaitForJobs              bool `yaml:"wait_for_jobs,omitempty" json:"wait_for_jobs,omitempty" jsonschema:"description=Whether to wait for all jobs to become ready,default=false"`
 
+	Labels map[string]string `yaml:"labels,omitempty" json:"labels,omitempty" jsonschema:"Labels that would be added to release metadata on sync"`
```

Comment:
I don't see purposes.

Reply:
So this option is added in both install and upgrade in helm 3.13.0, there is a test case which fails if we don't include or ignore all the fields that are new in a specific version, hence I have included this. If this is not needed I can put it in ignore as well, I was just thinking from a perspective that if even 1% is using this feature with helm, they would expect helmwave to also support it.